config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -112,9 +112,7 @@ def build_test(cloud='aws',
     if irsa and cloud == "aws" and scenario is None:
         if extra_flags is None:
             extra_flags = []
-        extra_flags.append(f"--override=cluster.spec.serviceAccountIssuerDiscovery.discoveryStore=s3://k8s-kops-prow/{job_name}/discovery") # pylint: disable=line-too-long
-        extra_flags.append("--override=cluster.spec.serviceAccountIssuerDiscovery.enableAWSOIDCProvider=true") # pylint: disable=line-too-long
-        extra_flags.append("--override=cluster.spec.iam.useServiceAccountExternalPermissions=true")
+        extra_flags.append(f"--discovery-store=s3://k8s-kops-prow/{job_name}/discovery") # pylint: disable=line-too-long
```

Comment:
I'm not sure we have to put the job name in there since `--discovery-store` appends the cluster name.

Reply:
Will update to append it.

Reply:
Updated to remove it.